packages/IngestAPI/src/main/resources/application.yml

```diff
@@ -15,9 +15,11 @@ server:
     enabled: true
     mime-types: application/json
 
-ingestApi:
-  usersJSON: '{"gw": "test-api-token"}'
+ingest-api:
```

Comment:
Since we've moved the api into a nested namespace of ingest, should the config here live under an `ingest` property?